src/DB_Command.php

```diff
@@ -1166,7 +1166,8 @@ function ( $a, $b ) use ( $order, $orderby ) {
 
 			// Display the rows.
 			$args = [
-				'format' => $format,
+				'format'     => $format,
+				'alignments' => [ 'Size' => Column::ALIGN_RIGHT ],
```

Comment:
Should we apply this only when `! WP_CLI\Utils\isPiped()`? If someone is piping table output to a script right now, this could break their parsing.

Also, this is currently blocked by https://github.com/wp-cli/php-cli-tools/pull/171